common/lib/xmodule/xmodule/js/spec/problem/edit_spec.coffee

```diff
@@ -231,28 +231,27 @@ describe 'MarkdownEditingDescriptor', ->
         [Explanation]
         """)
       expect(data).toXMLEqual("""<problem>
-        <multiplechoiceresponse>
-        <p>A multiple choice problem presents radio buttons for student input. Students can only select a single option presented. Multiple Choice questions have been the subject of many areas of research due to the early invention and adoption of bubble sheets.</p>
-        <p>One of the main elements that goes into a good multiple choice question is the existence of good distractors. That is, each of the alternate responses presented to the student should be the result of a plausible mistake that a student might make.</p>
-        <p>What Apple device competed with the portable CD player?</p>
-        <choicegroup type="MultipleChoice">
-            <choice correct="false">The iPad</choice>
-            <choice correct="false">Napster</choice>
-            <choice correct="true">The iPod</choice>
-            <choice correct="false">The vegetable peeler</choice>
-            <choice correct="false">Android</choice>
-            <choice correct="false">The Beatles</choice>
-        </choicegroup>
-        </multiplechoiceresponse>
-
-        <solution>
-        <div class="detailed-solution">
-        <p>Explanation</p>
-
-        <p>The release of the iPod allowed consumers to carry their entire music library with them in a format that did not rely on fragile and energy-intensive spinning disks.</p>
-
-        </div>
-        </solution>
+            <multiplechoiceresponse>
+                <p>A multiple choice problem presents radio buttons for student input. Students can only select a single option presented. Multiple Choice questions have been the subject of many areas of research due to the early invention and adoption of bubble sheets.</p>
+                <p>One of the main elements that goes into a good multiple choice question is the existence of good distractors. That is, each of the alternate responses presented to the student should be the result of a plausible mistake that a student might make.</p>
+                <p>What Apple device competed with the portable CD player?</p>
```

Comment:
I would prefer that we add some tests with `>><<` if they're not present already.